difflint/data/lint.sh

```diff
@@ -1,105 +0,0 @@
-#!/bin/bash
```

Comment:
As a general comment, I do think we should make sure pylint runs cleanly (within reason) on our Python code in this repo, although I could see making that a separate pull request.